app/src/main/java/com/example/cocktaildictionary/MainActivity.kt

```diff
@@ -16,6 +18,8 @@ import io.reactivex.rxjava3.schedulers.Schedulers
 class MainActivity : AppCompatActivity() {
 
     private lateinit var binding: ActivityMainBinding
+    private lateinit var cocktails: List<Cocktail>
```

Comment:
You don't need to save the list on a local variable, you can directly pass the results into the Adapter